bluephos/bluephos_pipeline.py

```diff
@@ -161,7 +211,10 @@ def get_pipeline(
     ap.add_argument("--input_dir", required=False, help="Directory containing input parquet files")
     ap.add_argument("--t_nn", type=float, required=False, default=1.5, help="Threshold for 'z' score (default: 1.5)")
     ap.add_argument("--t_ste", type=float, required=False, default=1.9, help="Threshold for 'ste' (default: 1.9)")
-    ap.add_argument("--no_xtb", action="store_false", dest="xtb", help="Disable xTB optimization (default: enabled)")
+    ap.add_argument("--t_dft", type=float, required=False, default=2.5, help="Threshold for 'ste' (default: 2.5)")
```

Comment:
should the `help` be `"Threshold for DFT energy diff"` or something, not `ste`?

Reply:
Yes, It's a typo